.gitattributes

```diff
@@ -0,0 +1,24 @@
+# Source files
+# ============
+*.pxd		  text eol=lf

```

Comment:
@seanlip 
@bbriggs has filed an issue for updating .gitattributes file
So this will be taken care in that PR
I am thinking of closing this PR and filing an issue for proxy handling only 
What you think?